third_party/terraform/tests/resource_dns_managed_zone_test.go.erb

```diff
@@ -128,6 +128,9 @@ resource "google_dns_managed_zone" "private" {
     networks {
       network_url = "${google_compute_network.%s.self_link}"
     }
+    dnssec_config {
+      state = "off"
```

Comment:
Is there a way to test this in a 'on' state? I'm not super familiar with the dnssec but this doesn't look like it tests that DNS can be configured by Terraform.

Reply:
Yeah, that's a good point. Just pushed up a (passing!) version of the test that has dnssec on.